addon/templates/components/nypr-o-header.hbs

```diff
@@ -2,14 +2,25 @@
 
 <div class="c-main-header__inner {{if this.outOfViewport 'c-floating-header is-visible'}}" id="header-inner">
 
-  {{yield (hash
-    menu=(component 'nypr-o-header/menu' donateURL=@donateURL close=(action toggleMenu false) aria-hidden=(if this.isOpen 'false' 'true') isOpen=this.isOpen)
-    left=(component 'nypr-o-header/left' openMenu=(action toggleMenu) ariaExpanded=(if this.isOpen 'true' 'false') ariaLabel=(if this.isOpen 'Close navigation menu' 'Open navigation menu'))
-    nav=(component 'nypr-o-header/nav' class="c-primary-nav--top" linkDataLabel='Top Nav')
-    right=(component 'nypr-o-header/right')
-    outOfViewport=this.outOfViewport
-    rules=this.rules
-  )}}
+  {{#if @showMenuButton}}
```

Comment:
If you put this conditional into the `nypr-o-header/left.hbs` template instead, you won't need to use two separate components or duplicate this block